udong-frontend/app/ui/feature/club/board/BoardContainer.tsx

```diff
@@ -1,13 +1,22 @@
 import { useRouter } from 'next/router'
+import { useState } from 'react'
 
 import { Spacer } from '../../../components/Spacer'
 import { HStack, VStack } from '../../../components/Stack'
 import { UdongButton } from '../../../components/UdongButton'
+import UdongLoader from '../../../components/UdongLoader'
 import { UdongSearchBar } from '../../../components/UdongSearchBar'
 import { PostItem } from '../../shared/PostItem'
 import { ScrollToTopButton } from '../../shared/ScrollToTopButton'
 
+const delay = (time: number) => {
+    return new Promise(resolve => setTimeout(resolve, time))
+}
+
 export const BoardContainer = () => {
+    const [loading, setLoading] = useState(true)
+    delay(600).then(() => setLoading(false))
```

Comment:
이렇게 하는게 더 간단하기도 하고 setLoading이 계속 호출되지 않아서 좋을거같아
```suggestion
    useEffect(() => {
        setTimeout(() => setLoading(false), 600)
    }, [])
```